students/JeroenSweerts/Lesson7/test_html_render.py

```diff
@@ -0,0 +1,167 @@
+import unittest
+import html_render as hr
+import os.path
+import io
+import sys
+from io import StringIO
+
+
+class TestHTMLRender(unittest.TestCase):
+
+    def test_Element_render(self):
+        f = StringIO()
```

Comment:
you could use `setUp` method to create `StringIO` instance so that it can be available in each test